tests/test_main.py

```diff
@@ -35,11 +35,29 @@ def test_process_with_valid_config_does_not_error(tidyDataFolders):
             "--config",
             "config.yml",
             "--file",
-            "solo_L2_mag-rtn-ll-internal_20240210_V00.cdf",
+            "imap_mag_l1a_norm-mago_20250502_v000.cdf",
         ],
     )
 
     print("\n" + str(result.stdout))
     assert result.exit_code == 0
     # check that file output/result.cdf exists
     assert Path("output/result.cdf").exists()
+
+
+def test_process_hk_ccsds(tidyDataFolders):
```

Comment:
Can I tempt you to look at the [bit about tests naming in the dev standards doc](https://imperialcollege.atlassian.net/wiki/spaces/PMLSD/pages/429555802/Developer+Standards) - would be good to try and get everyone being consistent at all possible.

> Consider test name structure [function_or_feature_under_test]_does_[some_single_action]_when_[this_input_is_satisfied] 
> e.g. generate_reports_creates_plot_of_absolute_field_when_at_least_24h_of_science_data_is_available.

Reply:
I've added a bit more context!